webpack.config.js

```diff
@@ -1,4 +1,25 @@
 const WebpackErrorNotificationPlugin = require('webpack-error-notification')
+const webpack = require('webpack')
+
+let apiHost
+
+const setupAPI = () => {
+  switch ( process.env.NODE_ENV ) {
+    case 'production':
+      apiHost = "'http://localhost:1337'"
+      break
+    case 'test':
+      apiHost = "'http://localhost:1337'"
+      break
+    case 'development':
+      apiHost = "'http://localhost:1337'"
+      break
+    default:
+      apiHost = "'http://localhost:1337'"
+  }
+}
+
+setupAPI()
```

Comment:
Lets refactor this setupAPI function to return the URL in each `case` instead of setting it to the `apiHost` variable. Then we can call `setupAPI()` on line 45:
```js
__HOST__: setupAPI()
```
Also, lets change the name of `setupAPI` to something like `determineHost`